src/main/resources/init_db.sql

```diff
@@ -7,3 +7,11 @@ CREATE TABLE `manufacturers` (
                                         `country` VARCHAR(225) NOT NULL,
                                         `is_deleted` TINYINT NOT NULL DEFAULT 0,
                                         PRIMARY KEY (`id`));
+
+CREATE TABLE `drivers` (
+                           `id` bigint NOT NULL AUTO_INCREMENT,
+                           `name` varchar(255) DEFAULT NULL,
+                           `license_number` varchar(255) DEFAULT NULL,
+                           `is_deleted` tinyint DEFAULT '0',
+                           PRIMARY KEY (`id`)
+) ENGINE=InnoDB AUTO_INCREMENT=6 DEFAULT CHARSET=utf8mb3;
```

Comment:
чому інкремент має починатись саме з 6?
```suggestion
) ENGINE=InnoDB  DEFAULT CHARSET=utf8mb3;
```